newswires/client/src/SearchSummary.tsx

```diff
@@ -212,12 +201,20 @@ const Summary = ({
 	} = query;
 
 	const displayCategoryCodes = (categoryCode ?? []).length > 0;
-	const displayExcludedCategoryCodes =
-		categoryCodeExcl && categoryCodeExcl.length > 0;
+	const displayExcludedCategoryCodes = (categoryCodeExcl ?? []).length > 0;
 	const displaySuppliers = (suppliers ?? []).length > 0;
+	const displayKeywords = (query.keyword ?? []).length > 0;
+	const displayExcludedKeywords = (query.keywordExcl ?? []).length > 0;
 
 	const displayFilters: boolean =
-		!!q || !!preset || displayCategoryCodes || displaySuppliers;
+		!!q ||
```

Comment:
It's very tempting to refactor this while I'm at it, but I don't _think_ there's a super clear alternative pattern, and didn't want the branch to get too bogged down